.github/workflows/test-nodejs.yaml

```diff
@@ -132,3 +133,62 @@ jobs:
           node secure_comparator_client.js
           kill -SIGTERM "$!"
           echo "ok"
+      - name: Test 'import' syntax
+        if: always()
+        run: |
+          cd $GITHUB_WORKSPACE/docs/examples/js/
+          echo "Test import syntax..."
+          node import_module.mjs
```

Comment:
just to be sure, did you tried some failed script to check some failure case to be sure that it's not always `echo ok`?

I found documentation for [run](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell) where described that by default `shell` used with `-e` flag. But real world can be unsynchronized from documentation and we met situations when tests ran but didn't exit with errors on failures.

Reply:
<img width="1317" alt="Screenshot 2023-11-28 at 12 55 54" src="https://github.com/cossacklabs/themis/assets/311543/83c34151-b68d-4d34-bd63-cd4337b32991">
Is it enough? Should I do a unit test?